Sources/Public/Animation/AnimationView.swift

```diff
@@ -259,7 +275,7 @@ final public class AnimationView: AnimationViewBase {
    */
   public var currentFrame: AnimationFrameTime {
     set {
-      removeCurrentAnimation()
+      removeCurrentAnimationIfNecessary()
```

Comment:
Calling `removeCurrentAnimation()` before `updateAnimationFrame(...)` is unnecessary for the Core Animation engine, but causes unnecessary animation setup passes (since it sets the `currentFrame` to `0`)